tailscale.tf

```diff
@@ -0,0 +1,8 @@
+# Generate Tailscale auth key
+resource "tailscale_tailnet_key" "this" {
+  reusable      = true
+  ephemeral     = true
+  preauthorized = true
```

Comment:
I would parameterize those 3 values as well, in case someone wants to customize it.